src/components/CattegoryButton.tsx

```diff
@@ -1,10 +1,17 @@
 type CategoryButtonProps = {
   category: string
+  active: string
 }
 
-export function CategoryButton({ category }: CategoryButtonProps) {
+export function CategoryButton({ category, active }: CategoryButtonProps) {
   return (
-    <div className='bg-light-grey flex justify-center w-fit py-1 px-4 m-1 items-center rounded-lg hover:bg-light-grey-hov text-light-blue font-bold text-sm cursor-pointer'>
+    <div
+      className={
+        active === category
```

Comment:
could you please use `clsx` as in here https://github.com/Antodi99/product-feedback/blob/main/src/components/Dropdown.tsx#L25